### PR TITLE
[tests] Upgrade testcontainers dependency to 1.16.2

### PR DESCRIPTION
**Pull request by wuchong**
Upgrade the testcontainers dependency from 1.15.1 to 1.16.2. This newer version should be quicker.

### CODE REVIEW

File: flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/utils/OracleCdcContainer.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        this.password = password;
+        return self();
+    }
+
+    @Override
+    public OracleCdcContainer withUrlParam(String paramName, String paramValue) {
+        throw new UnsupportedOperationException("The OracleDb does not support this");
+    }
+
+    @SuppressWarnings("SameReturnValue")
```
Comment (leonardBang) [inline flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/utils/OracleCdcContainer.java:114]: Do we need this ?

**Review by leonardBang — COMMENTED** [review]
Thanks @wuchong for the contribution, I left one minor comment

File: flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/utils/OracleCdcContainer.java (hunk 1 above)
Comment (wuchong) [reply flink-connector-oracle-cdc/src/test/java/com/ververica/cdc/connectors/oracle/utils/OracleCdcContainer.java:114]: I just copied the source code from `testcontainers:1.15.1`.

**Review by leonardBang — APPROVED** [review]
Thanks @wuchong for the great work, this will help reducing out test time, LGTM

**Pull request merged**